File: bmi.js
var weight=parseFloat(prompt("Enter weight:"));
var height=parseFloat(prompt("Enter height:"));
var bmi=weight / (height * height);

alert("Your BMI is: "+bmi);

if(bmi <= 18.5)
{
	alert("You're underweight.");
}
else if(bmi <= 25)
{
	alert("You're healthy.");
}
else if(bmi <= 30)
{
	alert("You're overweight.");
}
else
{
	alert("Holy McShizzle, you're freaking whale!");
}

File: bmi.R
Fbmi <- function(myW, myH){
	return(myW/(myH*myH))
}

w <- readline(prompt="Please enter you weight in KG ")
h <- readline(prompt="Please enter you height in M ")

bmi <- Fbmi(as.numeric(w), as.numeric(h))

if(bmi <= 18.5){
	print("You're underweight.")
} else if(bmi <= 25){
	print("You're healthy.")
} else if(bmi <= 30){
	print("You're overweight.")
} else{
	print("Holy McShizzle, you're freaking whale!")
}
